seed/grainlet/README.md

```diff
@@ -0,0 +1,6 @@
+# Grainlet Oscillator Example
+
+## Author
+
+Written by Emilie Gillet
+Ported by Ben Sergentanis
```

Comment:
I think this is a bit misleading. You wrote the example. She wrote the original source that the module was derived from. The proper attribution is already listed in the module's source code. 

If you want to mention that this is part of a part of the Plaits module in the example description, that's okay.